### DISCUSSION TITLE
Show file name in react-markdown code block

### DISCUSSION THREAD

**Discussion by rexwebmedia** [remarkjs/remark#1017] [Q&A] [answered]
I would like to show the file name in markdown code snippets. Any text after the code language is not shown in rendered html markup.

Here's the markup for markdown code block what I suggest.
<img width="520" alt="_" src="https://user-images.githubusercontent.com/63487995/179751154-1d887a1e-90cb-49c6-bf86-ffca81ba899a.png">

Here's the result I'm trying to achieve.
<img width="520" alt="utils_addTwoNumbers js" src="https://user-images.githubusercontent.com/63487995/179750651-b570e654-75ff-4dda-994d-d4d13601b861.png">

**Comment by Murderlon** [comment, marked as answer]
I would consider taking a look at some of the following plugins[^1]:

*   [`remark-code-blocks`](https://github.com/mrzmmr/remark-code-blocks) — select and store code blocks
*   [`remark-code-extra`](https://github.com/samlanning/remark-code-extra) — add to or transform the HTML output of code blocks (rehype compatible)
*   [`remark-code-frontmatter`](https://github.com/samlanning/remark-code-frontmatter)  — extract frontmatter from code blocks
*   [`remark-code-import`](https://github.com/kevin940726/remark-code-import) — populate code blocks from files
*   [`remark-code-screenshot`](https://github.com/Swizec/remark-code-screenshot) – turn code blocks into `carbon.now.sh` screenshots

[^1]: https://github.com/remarkjs/remark/blob/main/doc/plugins.md
